atom/browser/native_window.h

```diff
@@ -144,6 +148,10 @@ class NativeWindow : public content::WebContentsObserver,
                               const std::string& description) = 0;
   virtual void SetVisibleOnAllWorkspaces(bool visible) = 0;
   virtual bool IsVisibleOnAllWorkspaces() = 0;
+#if defined(OS_WIN)
```

Comment:
Now we prefer to have empty definition for platform-dependent APIs, so apps written for one specific platform won't throw errors on other platforms.

A few places still violated this rule and we are fixing them.